app/views/layouts/_chat_panel.html.slim

```diff
@@ -3,15 +3,13 @@ button.toggle-messages.btn#pollSlider-button
   Hide
 .d-messages.pollSlider
   h2 Chat
-  .messages-box
-    .messages-avatar-male
-    h4 Andrew
-    p Hi There...
-  .messages-box
-    .messages-avatar-female
-    h4 Sheyla
-    p Hi There...
-  button.messages-invite-friend.btn.btn-style Invite Friend
+  .related-block
+    - @users.each do |user|
+      .messages-box
+        .messages-avatar-male
+        h4 = user.name.capitalize
+        = link_to "conversation", "/messages/user/#{user.id}", class:('btn-style btn message-btn')
```

Comment:
Why do you need commented html here `/ button.messages-invite-friend.btn.btn-style Invite Friend` ?